ir/base/base.go

```diff
@@ -1,10 +1,19 @@
 // Package base implements the basic set of smart tags supported by a smart record.
 package base
 
-import "github.com/libp2p/go-smart-record/ir"
+import (
+	"github.com/libp2p/go-smart-record/ir"
+)
 
 // BaseGrammar is an assembler for the base vocabulary of smart tags supported by a record.
 var BaseGrammar = ir.SequenceAssembler{
 	// insert the assemblers of smart tags here
 	RecordAssembler{},
+	SyntacticAssembler{},
+}
+
+type SyntacticAssembler struct{}
+
+func (SyntacticAssembler) Assemble(ctx ir.AssemblerContext, src ir.Dict) (ir.Node, error) {
```

Comment:
@petar, this is probably wrong. I add it because with the current behavior of the `SequenceAssembler` I didn't know how to Assemble syntactic nodes.

Reply:
Good observation. I forgot to do this. I just pushed a commit on petar/p1 which fixes this. Just rebase.